addons/im_livechat/controllers/chatbot_script.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import is_html_empty, plaintext2html
+
+
+class LivechatChatbotScriptController(http.Controller):
+    @http.route('/im_livechat/chatbot_restart', type="json", auth="public", cors="*")
```

Comment:
why do you need to change the `cors`? is it called by another domain?

Reply:
The livechat (and theoretically the chatbot) can be embedded on another website.
You can see that all livechat routes in main.py also have the cors modified.